.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+dist: trusty
+
+language: python
+python: 3.5
+
+install:
+  # Install code sniffer.
+    pip install pycodestyle;
+    pip install --upgrade pycodestyle;
+    source ~/.bash_profile;
+    source ~/.bashrc;
+
+script:
+  # Run code sniffer on the scraper directory.
+    pycodestyle --show-source --max-line-length=119 --show-pep8 .
```

Comment:
@nirgn975 
 -  "tried to pull 2 commits to this PR" I was confiused, I ment 'tried to push 2 commits to this PR' and that failed.
 -  I wanted to delete commit to make the PR more orderly